src/main/java/core/basesyntax/FootballMatch.java

```diff
@@ -15,6 +15,13 @@
      * если не угадал ничего - возвращать 0.</p>
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+        int difference1 = result1 - bet1;
```

Comment:
Don’t forget to check yourself with checklist before submitting task(what about third point)
 https://mate-academy.github.io/jv-program-fulltime/01_core/checklist/hw_01.html

Reply:
Done